extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/exec/MSQReplaceTest.java

```diff
@@ -1913,7 +1931,8 @@ private CompactionState expectedCompactionState(
       Map<String, Object> context,
       List<String> partitionDimensions,
       List<DimensionSchema> dimensions,
-      GranularityType segmentGranularity
+      GranularityType segmentGranularity,
+      List<Interval> intervals
```

Comment:
all usages of this method seem to be passing a singleton list. So, we might as well just accept a single `Interval` here and make a singleton list out of it in this method itself.